sklearn/mixture/tests/test_dpgmm.py

```diff
@@ -89,10 +93,48 @@ def test_verbose_second_level():
             sys.stdout = old_stdout
 
 
+def test_digamma():
+    assert_warns_message(DeprecationWarning, "The function digamma is"
+                         " deprecated in 0.18 and will be removed in 0.20.",
+                         digamma, 3)
+
+
+def test_gammaln():
+    assert_warns_message(DeprecationWarning, "The function gammaln"
+                         " is deprecated in 0.18 and will be removed"
```

Comment:
If you look at the code by pressing 'View full changes', you'll see that the phrase 'Use scipy.special.digamma instead.' has been added.